### DISCUSSION TITLE
Maui Project Does Not Build, Apparent Version Skew

### DISCUSSION THREAD

**Discussion by plattski** [dotnet/maui#1414] [Q&A]
I have VS 2019, version 16.11.0 Preview 2. I installed Maui 5 Preview, and ran maui-check until it was happy. 

When I generate a Maui project using the VS workload, it doesn't compile. There are a number of error messages, all complaining "Unable to find package X".  When I look at the full text of the error message, they are all similar to this:

Error	NU1102	Unable to find package Microsoft.iOS.Runtime.iossimulator-x64 with version (= 14.5.100-preview.5.881)
  - Found 3 version(s) in nuget.org [ Nearest version: 14.5.100-preview.5.894 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages	MauiApp1	C:\Users\rollt\Source\Repos\MauiApp1\MauiApp1\MauiApp1\MauiApp1.csproj	1	
  
Any idea where I can find these files to build this app? Tx

**Comment by derekpapworth** [comment]
Hi @plattski 

I'm getting exactly the same error(s) with new install of VS2019 16.11 preview 2 plus following the documented install procedures and maui-check saying all ok (did notice it lists at the end references to .881 not .894 as per messages?) and I've tried uninstalling everything (VS, SDK, .net, etc) and then clean re-install but still the same.

The install procedure I've followed is as per documentation and a few YouTube examples where they all work so lost as to what I need to do to fix.

Have you found any answers since your post? Or any clues?

Anyone else have any ideas?

Thanks, Derek

**Comment by MarcusSchebesta** [comment]
Same exact issue here, installed everything according to the available documentation. maui-check reporting no issues.
Neither HelloMaui, nor freshly created projects builds with the errors described above.

**Comment by J-Dolezal** [comment]
For me helped updating maui check `dotnet tool update -g Redth.Net.Maui.Check` and then run it again
